code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -110,7 +110,7 @@
 	return
 
 /obj/item/weapon/gun/projectile/automatic/mini_uzi
-	name = "uzi"
+	name = "\improper 'Type U3' SMG"

```

Comment:
Well, apparently its bad adding more copyrighted names, and that counts as one of them apparently.